packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -898,10 +898,19 @@ const tests = {
           const local1 = 42;
           const local2 = '42';
           const local3 = null;
+          const local4 = false;
```

Comment:
nit: could do with a refactor?  How about popping the values in an array.  Then
`
const locals = [ 42, '42', null, false, undefined, Symbol(42), BigInt(42)];
locals.forEach(console.log);
`